### ISSUE TITLE
no-unused-vars false positive in case of address payable argument without name

### ISSUE THREAD

**Issue by ZumZoom** [protofire/solhint#250]
```solidity
contract C {
    function foo(address payable /* addr */) public view {
        this;
    }
}
```

This contract triggers `Variable "payable" is unused` however payable is not a variable.

**Comment by dbale-altoros** [comment]
should be closed with this PR
https://github.com/protofire/solhint/pull/399
